### ISSUE TITLE
Output Accuracy problem

### ISSUE THREAD

**Issue by jimd6776** [hls-fpga-machine-learning/hls4ml#197]
Hi,
I have synthesized successfully a CIFAR type model with 16-bit weights and activations. The model has ~80% accuracy when tested on keras. However, in FPGA the output results differ even when a class is 99% predicted in SW. Is this expected? Shouldn't the results be similar in this case?

Thanks in advance

**Comment by thesps** [comment]
It depends. It may be that the 16-bits used don't cover the full range of the weights/activations of the model. Have you tried the model profiling utilities? See: https://fastmachinelearning.org/hls4ml/PROFILING.html 
Also, the next release will have some new features which will make it easier to track down where discrepancies arise. 
If you point me to the model file and configuration online I can see if there's anything obvious.
